### DISCUSSION TITLE
Where does the "new-page-template.md" belong?

### DISCUSSION THREAD

**Discussion by robkle153** [google/docsy#1652] [Q&A]
The documentation says that the template for creating a new page belongs in the folder asset/stubs/. This does not work for me. Do you know where it has to go?

![image](https://github.com/google/docsy/assets/139347669/edc47992-0eec-4c7a-8cd7-680ff1447a79)

I only get something like this:

![image](https://github.com/google/docsy/assets/139347669/bb03ef6b-6dc7-4d93-aa4c-0e7628e12f56)

Thanks for you help!

**Comment by deining** [comment]
Following the instructions you mentioned above, everything works perfectly fine for me:

![grafik](https://github.com/google/docsy/assets/18169566/db9d42e6-d595-4cb5-919a-8c38f625f541)

Is your repo publicly available? If not, can you tell us the URL shown in your browser when creating a new page?

**Reply by robkle153** [reply]
Sorry its not public.

https://.../-/new/main/content/en/Docs 

This is the URL. When I use my terminal to create a new file with `hugo new Docs/new-page.md ` it works perfectly with my templates in the archetypes folder (docs.md for documentations and blog.md for blog posts), but not with the repo links on the right menu. Maybe I forgot smthg in my hugo.toml.

```
github_repo = "https://..."
github_branch = "main"
github_subdir = ""
gcs_engine_id = "d72aa9b2712488cc3"
algolia_docsearch = false
offlineSearch = false
```
